include/c3d/effect.h

```diff
@@ -1,15 +1,80 @@
+/**
+ * @file effect.h
+ * @brief Configure GPU state
+ */
 #pragma once
 #include "types.h"
 
+/**
+ * @brief Specify mapping of depth values from normalized device coordinates to window coordinates.
+ * @param[in] bIsZBuffer Enables or disables depth range. The initial value is true.
+ * @param[in] zScale     Specifies mapping of depth values from normalized device coordinates to window coordinates (nearVal - farVal). The initial value is -1.0f.
+ * @param[in] zOffset    Sets the scale and units used to calculate depth values (nearVal + polygonOffset). The initial value is 0.0f.
+ */
 void C3D_DepthMap(bool bIsZBuffer, float zScale, float zOffset);
+
+/**
+ * @brief Specify whether front-facing or back-facing facets can be culled.
```

Comment:
nit: `facets` -> `faces` ?

Reply:
Afaict, facet is correct here:
https://en.wikipedia.org/wiki/Facet_(geometry)

This seems to be where I got the description from:
https://docs.microsoft.com/en-us/windows/win32/opengl/glcullface

Reply:
Huh, today I learned. From that wikipedia:
> In three-dimensional geometry, they are often called "faces" without qualification.[[4]](https://en.wikipedia.org/wiki/Facet_(geometry)#cite_note-4)

So I guess both are valid, I've just seen "face" used more often, I guess.